src/pages/elements/Namespaces.js

```diff
@@ -97,9 +113,9 @@ function Namespaces({ namespaces }) {
   };
 
   const tableCols =
-    typeof namespaces[0].cluster === 'undefined'
-      ? [colName, colPath, colDescription]
-      : [colName, colPath, colCluster, colDescription];
+    typeof namespaces[0].cluster.name === 'undefined'
```

Comment:
the reason for this change is:
https://github.com/app-sre/visual-qontract/pull/2/files#diff-8f8318b35d22a5e1ddf420d0b5df418eR23